csunplugged/templates/resources/treasure-hunt.html

```diff
@@ -0,0 +1,36 @@
+{% extends "resources/base-resource.html" %}
+
+<!-- Name of resource -->
+{% block name %}{{ resource.name }}{% endblock name %}
```

Comment:
happy with opening and closing a block on the same line? Or should there be a new line here for consistency? (see lines 7 and 8)